### ISSUE TITLE
No backoffice dashboard since Actix

### ISSUE THREAD

**Issue by tpayet** [meilisearch/MeiliSearch#707]
Received by email:

Bonjour!

I'm new to Rust and may be doing something wrong, but I need Your help.

Following the instruction about how to build MeiliSearch from source (under MasOS X Catalina) I get bin file without any errors. But I can't use the web form, only from command line (remember about the instruction for 'development' mode).I see some misbehavior in log after starting MeiliSearch: 
```
[2020-05-17T11:59:12Z INFO  actix_server::builder] Starting 16 workers
[2020-05-17T11:59:12Z INFO  actix_server::builder] Starting "actix-web-service-127.0.0.1:7700" service on 127.0.0.1:7700
```

But, then I download compiled binary from Your github, I get another view:
```
[2020-05-17T12:01:54Z INFO  tide::server] Server is listening on: http://127.0.0.1:7700
```
And all works just fine. Cli or web form, it doesn't matter.

I attached two screenshots to the letter, on one the output of the command of my assembly (0010), and on the second (0001) the output of the binary from your github.
Could you please get me some advice in my process?

<img width="1378" alt="0001" src="https://user-images.githubusercontent.com/10537452/82667835-e6a69580-9c38-11ea-8f02-bbaa1a761557.png">
<img width="1378" alt="0010" src="https://user-images.githubusercontent.com/10537452/82667842-eb6b4980-9c38-11ea-8564-ceb0f36157cc.png">

**Comment by Kerollmops** [comment]
I think this issue is fixed by #695, I will merge it today I hope.

**Comment by eugengo** [comment]
Hello!
It's my issue. Will report about it after fix.
